lib/taurus/qt/qtgui/taurusgui/taurusgui.py

```diff
@@ -833,9 +833,6 @@ def createCustomPanel(self, paneldesc=None):
             w.setCustomWidgetMap(self.getCustomWidgetMap())
         if paneldesc.model is not None:
             w.setModel(paneldesc.model)
-        if isinstance(w, TaurusBaseComponent):
```

Comment:
Removing this would break applications that rely on the user to be able to modify the custom widgets after creation. For example, consider the following pretty common use case: a user clicks on "Panel-->New Panel --> selects a TaurusForm and gives it a name -->accepts the dialog --> attempts to drag and drop values from other forms.

Reply:
I am preparing some modifications to the code and I'll take care of this, so do not act on this for now.